web/libs/editor/src/stores/Annotation/Annotation.js

```diff
@@ -308,8 +308,22 @@ const _Annotation = types
       });
     },
 
+    get isNonEditableDraft() {
+      const isKnownUsers = !!self.user && !!self.store.user;
+      // If we do not know what user created draft
+      // and who we are, then, we shouldn't prevent the ability to edit annotation
+      // because we can't predict is it our draft or not.
+      // It most probably could be relevant for standalone `lsf`
+      if (!isKnownUsers) return false;
```

Comment:
while i approved this - im wondering if we should override this in LSE to not allow it if user is unknown

Reply:
From my understanding in LSE we always should have an information about users. It's a part under our control.